pkgs/by-name/mi/microfetch/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage {
+  pname = "microfetch";
+  version = "unstable-2024-08-04";
+
+  src = fetchFromGitHub {
+    owner = "NotAShelf";
+    repo = "microfetch";
+    rev = "72d3130aa098a9e99dbca264a96e2372a1740204";
+    hash = "sha256-4l5EWjvueSF4ujNvW867HwVO/YoW3e1WLE7f+qlJ0aE=";
+  };
+
+  cargoHash = "sha256-d0S51K8f7w3tjDJMXinKKpxQKs3jgFFC0fhP/VDrU8I=";
+
+  meta = {
+    description = "A microscopic fetch script in Rust, for NixOS systems";
```

Packaging guidelines dictate that description cannot start with indefinite articles.
```suggestion
    description = "Microscopic fetch script in Rust, for NixOS systems";
```

---

I've also updated the repo description, since it doesn't make too much of a difference.

---

Just pushed the change.

On a side note: Is there a way to accept suggestions without creating a separate commit, i.e. amend the most recent commit?

---

Not to my knowledge.